3.9/graphs-smart-graphs-getting-started.md

```diff
@@ -0,0 +1,156 @@
+      var graph_module = require("@arangodb/smart-graph");
+      var graph = graph_module._create("myGraph", [], [], {smartGraphAttribute: "region", numberOfShards: 9, isDisjoint: true});
+      graph_module._graph("myGraph");
+     ~graph_module._drop("myGraph");
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock smartGraphCreateGraphHowToDisjoint1_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+## Add vertex collections
+
+This is analogous to General Graphs. Unlike with General Graphs, the
+**collections must not exist** when creating the SmartGraph. The SmartGraph
+module will create them for you automatically to set up the sharding for all
+these collections correctly. If you create collections via the SmartGraph
+module and remove them from the graph definition, then you may re-add them
+without trouble however, as they will have the correct sharding.
+
+{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+    @startDocuBlockInline smartGraphCreateGraphHowTo2_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{smartGraphCreateGraphHowTo2_cluster}
+     ~var graph_module = require("@arangodb/smart-graph");
+     ~var graph = graph_module._create("myGraph", [], [], {smartGraphAttribute: "region", numberOfShards: 9});
+      graph._addVertexCollection("shop");
+      graph._addVertexCollection("customer");
+      graph._addVertexCollection("pet");
+      graph_module._graph("myGraph");
+     ~graph_module._drop("myGraph", true);
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock smartGraphCreateGraphHowTo2_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+## Define relations on the Graph
+
+Adding edge collections works the same as with General Graphs, but again, the
+collections are created by the SmartGraph module to set up sharding correctly
+so they must not exist when creating the SmartGraph (unless they have the
+correct sharding already).
+
+{% arangoshexample examplevar="examplevar" script="script" result="result" %}
+    @startDocuBlockInline smartGraphCreateGraphHowTo3_cluster
+    @EXAMPLE_ARANGOSH_OUTPUT{smartGraphCreateGraphHowTo3_cluster}
+     ~var graph_module = require("@arangodb/smart-graph");
+     ~var graph = graph_module._create("myGraph", [], [], {smartGraphAttribute: "region", numberOfShards: 9});
+     ~graph._addVertexCollection("shop");
+     ~graph._addVertexCollection("customer");
+     ~graph._addVertexCollection("pet");
+      var rel = graph_module._relation("isCustomer", ["shop"], ["customer"]);
+      graph._extendEdgeDefinitions(rel);
+      graph_module._graph("myGraph");
+     ~graph_module._drop("myGraph", true);
+    @END_EXAMPLE_ARANGOSH_OUTPUT
+    @endDocuBlock smartGraphCreateGraphHowTo3_cluster
+{% endarangoshexample %}
+{% include arangoshexample.html id=examplevar script=script result=result %}
+
+## Using SatelliteCollections in SmartGraphs
```

Comment:
Headlines should have a blank line above and below. Some Markdown renderers don't recognize them otherwise, but it's also easier to spot headlines this way.

```suggestion
## Using SatelliteCollections in SmartGraphs

```